app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -159,4 +159,22 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
         EitherT.pure[Future, String, UserInfo](UserInfo.AlreadyHasAccount(value))
 
     }
+
+  def createAccount: Action[AnyContent] = authorisedFromStride { implicit request ⇒
+    checkSession(SeeOther(routes.StrideController.getEligibilityPage().url),
+                 checkIsEligible(result ⇒ helpToSaveConnector.createAccount(result.nSIUserInfo)
+        .fold(
+          error ⇒ {
+            logger.warn(s"error during create account call, error: $error")
+            InternalServerError
+          }, {
+            case AccountCreated ⇒
+              Ok(routes.StrideController.getAccountCreatedPage().url)
```

Comment:
`Ok(views.xxxxxx.html)` and remove the GET from the app.routes file

Reply:
and check the coverage by running `sbt clean compile coverage test coverageReport`